src/components/ebay-split-button/split-button.stories.js

```diff
@@ -0,0 +1,227 @@
+import { tagToString } from '../../../.storybook/storybook-code-source';
+import { addRenderBodies } from '../../../.storybook/utils';
+import button from './index.marko';
+import readme from './README.md';
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+    // input: {
+    //     ...args,
+    //     spread: null,
+    //     ...args.spread,
+    //     renderBody(out) {
+    //         out.html(args.renderBody);
+    //     },
+    // },
+});
+// const Template = args =({ input: withRenderBody(args) })
+
+export default {
+    title: 'ebay-split-button',
+    component: button,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        renderBody: 'Button',
+        href: {
```

Comment:
I tried adding `href` to component and it doesn't seem to be honoured.

Reply:
We will need additional styles to get href to work. Skin is missing styles for anchor split button. Will add it to https://github.com/eBay/skin/issues/1690

Reply:
I see. Thanks!